.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+# This script is executed whenever the docker container is (re)started.
+
+#===============================================================================
+# debugging
+set -x
+
+#===============================================================================
+# environment
+reentry scan
+
+export SHELL=/bin/bash
+
+#===============================================================================
+# setup AiiDA
+aiida_backend=django
+
+if [ ! -d /home/$SYSTEM_USER/.aiida ]; then
+    verdi setup                                \
+        --profile default                      \
+        --non-interactive                      \
+        --email some.body@xyz.com              \
+        --first-name Some                      \
+        --last-name Body                       \
+        --institution XYZ                      \
+        --db-backend $aiida_backend            \
+        --db-username aiida                    \
+        --db-password aiida_db_passwd          \
+        --db-name aiidadb                      \
+        --db-host localhost                    \
+        --db-port 5432                         \
+        --repository /home/$SYSTEM_USER/aiida_repository
```

Comment:
That's a bit of a political question - should there be a pre-configured profile by default?

The upside is that people can directly start running AiiDA commands,
the downside is they won't bother to set up their own profile and thus not learn how to do it, and when they start exporting data they may discover that the User name/email/... is not what they want.
Also, it is not needed for running tests.

I would lean towards not setting it up by default but I can see arguments for both use cases. 
I would suggest to:

 * Use an `ENV SETUP_DEFAULT_PROFILE 0` in the Dockerfile and check the value of this variable in the `configure-aiida.sh` script. One can then start a container with `docker run ... -e SETUP_DEFAULT_PROFILE=1` and it will create the default profile.

 * for the profile setup, (re)use a [config yaml file](https://github.com/aiidateam/aiida-core/blob/develop/.github/config/profile.yaml) with the `--config` option. if this yaml is present on the container, it means one can set up the profile in one line, even if it's not yet there

 * please check that `verdi quicksetup` works out of the box.

Reply:
> Use an `ENV SETUP_DEFAULT_PROFILE 0`
agreed

> or the profile setup
For the moment I will still the current way, but make everything configurable through `ENV`. That way we can preconfigure things for AiiDA lab.

Reply:
> please check that `verdi quicksetup` works out of the box.

it works.